src/domain/accounts/domain/src/entities/account.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use chrono::{DateTime, TimeZone, Utc};
+use lazy_static::lazy_static;
+use opendatafabric::{AccountID, AccountName};
+use serde::{Deserialize, Serialize};
+
+///////////////////////////////////////////////////////////////////////////////
+
+// TODO: have some length restrictions (0 < .. < limit)
+pub type AccountDisplayName = String;
+
+pub const DEFAULT_ACCOUNT_NAME_STR: &str = "kamu";
+pub const DEFAULT_AVATAR_URL: &str = "https://avatars.githubusercontent.com/u/50896974?s=200&v=4";
+pub const DEFAULT_EMAIL: &str = "kamu@example.com";
```

Comment:
I really wish we got rid of these as per our discussion about not making `kamu` account special.

We could at least move them to layers where they make sense (CLI, tests) - don't think these belong in `domain`.

Reply:
There is a serious dependency on the  `DEFAULT_ACCOUNT_NAME` at the level of dataset repositories and OSO authorizer. Basically, we have to put something when there is no account name in a dataset alias.

Reply:
I got rid of email completely (not needed much) and avatar (moved to CLI).
But id & name are integral part of single-tenant alias resolutions.

Reply:
`kamu` as the default account itself is already problematic - we don't want to make it look like `kamu` "owns" all datasets in ST mode. This account only makes sense on our own environments. In CLI its better to use the OS username, in ST api-server - to make it configurable.

I think the problem is that on dataset repo level - single-tenant means no account at all, but in auth and GQL we decided to have accounts in both configurations and now its on them to provide defaults. But they are not the right layer to decide what default account should be...

We can deal with this later, as it existed before this PR, but I think this IS a problem and we should ticket it up as tech debt.

Reply:
> I think the problem is that on dataset repo level - single-tenant means no account at all, but in auth and GQL we decided to have accounts in both configurations and now its on them to provide defaults. But they are not the right layer to decide what default account should be...

We can't let UI decide either, neither we can have separate UIs. This affects Angular URLs design a lot.